source/includes/documentation/_vue_js_elements.md

```diff
@@ -2964,3 +2964,62 @@ Available methods in Java JDI Light:
 
 For examples of usage see: [JDI Vuetify Chip tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/ChipsTests.java).
 
+### 5.40 Badge
+
+[Badge Vuetify documentation page](https://vuetifyjs.com/en/components/badges/)
+
+Badge is located in the following class:
+
+- __Java__: _com.epam.jdi.light.vuetify.elements.common.Badge.java_
+
+```java
+    @Test
+    public void typeBadgesTest() {
+        lockUnlockAccountBadge.is().icon();
+        dotBadge.has().notIcon();
+        imageBadge.has().avatar();
+        dotBadge.has().notAvatar();
+        dotBadge.is().dot();
+        imageBadge.is().notDot();
+    }
+
+    @Test
+    public void textBadgesTest() {
+        Badge simpleBadge = simpleBadges.get(1);
+        simpleBadge.show();
+        simpleBadge.has().text("1");
+    }
+```
+
+__Badges__ - The `v-badge` component superscripts or subscripts an avatar-like icon or text onto content to highlight information to a user or to just draw attention to a specific element.
+Content within the badge usually contains numbers or icons.
+
+![Badges example](../../images/vuetify/badges.png)
+
+Available methods in Java JDI Light:
+
+|Method | Description                       | Return Type
+--- |-----------------------------------| ---
+**is()** | Badge assert                      | BadgeAssert
+**badge()** | Gets '{name}' badge               | UIElement
+**isBordered()** | Checks that '{name}' is bordered  | boolean
+**isInline()** | Checks that '{name}' is inline    | boolean
+**isBottom()** | Checks that '{name}' is bottom    | boolean
+**isOverlap()** | Checks that '{name}' is overlap   | boolean
+**isDot()** | Checks that '{name}' is dot       | boolean
+**isAvatar()** | Checks that '{name}' is avatar    | boolean
+**backgroundColor()** | Gets '{name}' background color    | String
+**image()** | Gets '{name}' image               | Image
+**isDisplayed()** | Checks that '{name}' is displayed | boolean
+**icon()** | Gets '{name}' icon | Icon
+**hasIcon()** | Checks that '{name}' has icon     | boolean
+**getText()** | Gets '{name}' badge text          | String
+**isTile()** | Checks that '{name}' is tile | boolean
+**hasLeftAlignment()** | Checks that '{name}' has left alignment | boolean
+**hasRightAlignment()** | Checks that '{name}' has right alignment | boolean
+**color()** | Gets '{name}' color | String
+**backgroundColor()** | Gets '{name}' background color | String
+**image()** | Gets '{name}' image | Image
+
+For examples of usage see:
+[JDI vuetify page tests for Badges](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/BadgesTests.java)
```

Comment:
Hi :) failed to find this point "7.	Element code (with reference to framework version - IMPORTANT!)", please, clarify